### PR TITLE
Adding LightAgentConfigurationType.createdAt

### PR DESCRIPTION
**Pull request by lasryaric**
## Description
Adding LightAgentConfigurationType.createdAt
There is a follow up PR coming with Amplitude.trackAssistantCreated().

### CODE REVIEW

File: front/lib/api/assistant/configuration.ts [hunk 1]
```diff
@@ -890,6 +891,7 @@ export async function createAgentConfiguration(
      */
     const agentConfiguration: AgentConfigurationType = {
       id: agent.id,
+      createdAt: agent.createdAt.getTime(),
```
Comment (philipperolet) [inline front/lib/api/assistant/configuration.ts:894]: it seems the same thing as "versionCreatedAt" right below, why would we need 2 fields?
Also, if what you needed was the time at which the agent was first created, then this won't be it, you need to fetch version 0.

**Review by philipperolet — CHANGES_REQUESTED** [review]
Hey ! Something to clarify before moving forward, we can chat after lunch if needed

File: front/lib/api/assistant/configuration.ts (hunk 1 above)
Comment (lasryaric) [reply front/lib/api/assistant/configuration.ts:894]: Just did not see it. Closing. Thanks.

**Comment by lasryaric** [comment]
Useless.

**Pull request closed without merging**